plutus-ir/test/recursion/even3Eval.golden

```diff
@@ -1,5 +1,7 @@
 (abs
-  out_Bool_19
+  out_Bool_103
   (type)
-  (lam case_True_20 out_Bool_19 (lam case_False_21 out_Bool_19 case_False_21))
+  (lam
+    case_True_104 out_Bool_103 (lam case_False_105 out_Bool_103 case_False_105)
+  )
```

Comment:
It's a little strange that uniques are much greater now.

Reply:
Yeah, not sure why that is.